package.json

```diff
@@ -99,7 +102,7 @@
     "wait-on": "5.0.0",
     "webpack": "4.43.0",
     "webpack-cli": "3.3.11",
-    "webpack-dev-server": "3.10.3"
+    "webpack-dev-server": "^3.11.0"
```

Comment:
I'm glad we upgrade, but same, exact version :grin:

Reply:
Done